english/pinch/go-out/keep-running/keep-running.md

```diff
@@ -1,3 +1,10 @@
 You keep running, going nowhere in particular.
 
-Then you tripped over something and fell flat on your face.
\ No newline at end of file
+Then you tripped over something and fell flat on your face.
+
+You tripped on a half-open manhole cover. You move the cover to 
+reveal the top of a ladder leading down into the dark.
+
+[Climb down](../../../zork/zork.md)
+
+[Notice that there's a patch of fluffy moss off to your left and go to take a nap?](english/explore-outside/moss-nap/moss-nap.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.